recipe/meta.yaml

```diff
@@ -7,48 +12,48 @@ package:
 source:
   fn: mpfr-{{ version }}.tar.bz2
   url: http://ftp.gnu.org/gnu/mpfr/mpfr-{{ version }}.tar.gz
-  sha256: ae26cace63a498f07047a784cd3b0e4d010b44d2b193bab82af693de57a19a78
-  patches:
-  - patches/build-vc14.patch
```

Comment:
You should remove these patch files from the repo if they are not needed anymore.